docs/internals.md

```diff
@@ -25,7 +25,7 @@ The code is statically analyzed with [Psalm](https://psalm.dev/). To run static
 ./vendor/bin/psalm
 ```
 
-## Code style
+## Rector
```

Comment:
2 different tools are now mixed up under 1 header.

Reply:
Original is like this:
      code style
      Use Rector to make codebase follow some specific rules or use
      either newest or any specific version of PHP:
      ./vendor/bin/rector

You are referencing the rector. I think it would be appropriate for the title to be the same as in https://github.com/yiisoft/yii-dataview/blob/master/README.md#rector.

All the references I saw were to the rector. I don't understand your doubt. Could you explain which "two tools" you are referring to?

Reply:
Composer Require Checker is completely different tool and the info about it is now under Rector header, which is wrong.

Reply:
I researched ## Code style and ## Rector and understood the difference. Unfortunately I made a lot of changes about this. But I already found a way to fix these changes. Thanks for paying attention.